_posts/2017-3-26-armory-and-hard-forks.md

```diff
@@ -0,0 +1,47 @@
+
+Although the Armory developers do not support such hard forks, the Armory wallet software does not perform any consensus checks as it relies on its connection to a local Bitcoin node. This node is typically Bitcoin Core, but anything based upon that will work as well; this includes Bitcoin Unlimited. Thus should a hard fork occur, so long as no changes are made to the transaction or block format, Armory will be compatible with the hard fork and will allow users to continue to transact on the forked network.
+
+Because transactions for the forked network and for the original will primarily be valid on both networks, there is the risk for replay attacks to occur and for the loss of coins due to transaction replay. The following will help users avoid transaction replay should Bitcoin Unlimited or another similar block size increase hard fork proposal be activated. However this procedure is not necessarily guaranteed to work and assumes that the forked chain will have significantly lower transaction fees than the original chain.
+
+Should a chain fork occur, more up-to-date information about what Armory users can do will be posted on the [Armory forums](https://bitcointalk.org/index.php?board=97.0) and on this website.
+
+### The setup
+
+The Armory database engine builds off of on-disk blockchain data and does not enforce consensus rules. In other words, any client that uses the same block data format and magic word as Bitcoin Core is compatible with Armory. From your perspective, running Armory against the BU chain will show you your BU balance, and vice versa. 
+
+In practice, this means that to be able to interchange seamlessly between your Core and BU transaction history, you need 2 copies of the blockchain data; one for the Core chain and one for the BU chain. Same goes with the Armory database, as it is intimately tied to the underlying block data. However, you do not need to rescan the BU chain to get the copy of pre-fork Core database to work off of it. To be absolutely safe, you should use rebuild and rescan the databases for both chains after the fork, and even fully redownload the blockchain to ensure that everything is correct.
+
+To summarize, you need:
+
+- 1 copy of the blockchain to run exclusively against the Core client, with the associated Armory database directory.
+- 1 copy of the blockchain to run exclusively against the BU client, with associated Armory database directory.
+
+You can swap between the two by pointing Armory to the relevant pair of block data folder and database folder by using the `--satoshi-datadir` and `--dbdir` command line options respectively.
+
+### Avoiding Transaction Replay
+
+A transaction replay attack is where one transaction can end up confirmed in both blockchains thus resulting in coins on one chain being spent when you did not want those to be spent. This can happen either accidentally or intentionally where someone broadcasts your transaction to both networks.
```

Comment:
"A transaction replay attack is where one transaction can end up confirmed in both blockchains thus resulting in coins on one chain being spent when you did not want those to be spent. This can happen either accidentally or intentionally where someone broadcasts your transaction to both networks."

A transaction replay...both blockchains, thus resulting.... This can happen accidentally or intentionally due to someone broadcasting your transaction on both networks.